test/conftest.py

```diff
@@ -8,12 +8,9 @@
 DATAFILES_PATH = os.path.join(HERE, 'data')
 
 
-# after we have run all tests, use git to delete the built files in data/
-
-
-@pytest.fixture(scope='session', autouse=True)
+@pytest.fixture(autouse=True)
```

Comment:
Not making this session level will slow down the tests considerably due to compilation times. Was this change motivated by something being incorrect in the other tests?

Reply:
Yes, it looks like the tests have become pretty slow. I changed it to function-level because I had some non-deterministic tests depending on the order in which I ran them. Will try reverting this change and get the tests to pass.